CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ All notable changes to AET will be documented in this file.
 ## Unreleased
 **List of changes that are finished but not yet released in any final version.**
 
+- [PR-314](https://github.com/Cognifide/aet/pull/314) Updated osgi annotation in jobs
```

Comment:
Please group all those PRs in a single point.
And also change the target of this PR to `feature/upgrade-osgi-annotations`.